dependencies.yaml

```
@@ -734,7 +734,7 @@ dependencies:
         matrices:
           - matrix: {cuda: "12.*"}
             packages:
-              - &pytorch_pip torch>=2.0,<2.2.0a0
+              - &pytorch_pip torch>=2.0,<2.4
```

```suggestion
              - &pytorch_pip torch>=2.0,<2.4.0a0
```

I think the alpha spec should be preserved here, to avoid accidentally pulling in alpha releases of `torch` 2.4